minesweeper.py

```diff
@@ -187,6 +162,19 @@ def update_statusbar_mines_left(self):
         """ Prints out how many mines are left """
         statusbar_count["text"] = f"{self.mine_count - len(self.flag_dict)} mines left"
 
+def draw_image(x, y, image):
+    if image == flag_image or image == wrong_flag_image:
+        return canvas.create_image(
+        x * button_size + (button_size / 2),
+        y * button_size + (button_size / 2),
+        image=image,
+        anchor="center")
+    else:
+        return canvas.create_image(
+        int(x * button_size),
+        int(y * button_size),
```

Comment:
Just noticed that these `int()`s are not necessary:

```suggestion
        x * button_size,
        y * button_size,
```